themes/generic/src/main/assets/js/groups.js

```diff
@@ -128,9 +122,8 @@ $(function() {
       .val(group);
 
     // Inline snippets:
-    for (var i = 0; i < catalog[supergroup].length; i++) {
-      var peer = catalog[supergroup][i];
-      if (peer == group) {
+    catalog[supergroup].forEach(peer => {
```

Comment:
I'm never sure if such things are available on all browsers we care about :D - :+1: I guess

Reply:
Me too. 😅 

But, yeah, it is widely supported according to https://caniuse.com/mdn-javascript_builtins_array_foreach.